src/google/cloud/ndb/metadata.py

```diff
@@ -223,61 +225,140 @@ def key_to_property(cls, key):
             return key.id()
 
 
-class EntityGroup(_BaseMetadata):
-    """Model for __entity_group__ metadata, available in HR datastore only.
```

Comment:
I wonder what HR means, here, and if it's relevant to us?

Reply:
I cut it here. The datastore says the query for this entity is unsupported, so I made the constructor throw a NoLongerImplementedError. If it turns out later that the HR is something that needs to be taken into account, we can bring this back.